src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -178,13 +178,13 @@ def static_dag(job, uuid, rg_line, inputs):
             'dir_suffix': inputs.dir_suffix}
 
     # get head BWA alignment job function and encapsulate it
-    bwa = job.wrapJobFn(download_shared_files,
+    inputs.rg_line = rg_line
+    inputs.output_dir = 's3://{s3_bucket}/alignment{dir_suffix}'.format(**args)
+    bwa = job.wrapJobFn(download_reference_files,
                         inputs,
                         [uuid,
                          's3://{s3_bucket}/{sequence_dir}/{uuid}_1.fastq.gz'.format(**args),
-                         's3://{s3_bucket}/{sequence_dir}/{uuid}_2.fastq.gz'.format(**args)],
-                        's3://{s3_bucket}/alignment{dir_suffix}'.format(**args),
-                        rg_line).encapsulate()
```

Comment:
> Why isn't rg_line just in inputs to start with? 

Looking at the code, I'm not sure if it's even defined anywhere in `align_and_call`. 

`rg_line` gets set from `uuid_items[1]`, which is taken from `uuid_rg` from `uuid_list`, but `uuid_list` is parsed from the manifest which has no entry for read group...  (confused yet?).